utils/src/activityLog/activities/ExecuteActivity.ts

```diff
@@ -24,6 +24,12 @@ export class ExecuteActivity<TContext extends types.ExecuteActivityContext = typ
         }
     }
 
+    protected override report(progress?: { message?: string; increment?: number }): void {
+        const message: string | undefined = this.context.activityChildren?.length ? this.timerMessage : progress?.message;
+        this._onProgressEmitter.fire({ ...this.getState(), message });
```

Comment:
Made a new change here.  I decided to override the report method in the concrete class.   Two reasons for this:
1) The concrete class is what actually holds the `ExecuteActivityContext`
2) The abstract class base implementation doesn't seem coupled to the Wizard, so I didn't want to pollute its implementation with Azure Wizard-y things.

The way this behaves:
- If an activity has children, we just show a timer as the description.  We offload the responsibility of showing what's being executed to the children
- If no children, we still should indicate what's happening, so let's default back to any progress report messages in the description